test/api/peers.js

```diff
@@ -348,6 +348,32 @@ describe('GET /api/peers', function () {
 		});
 	});
 
+    it('using orderBy == "height:desc" should not place NULLs first', function (done) {
+        var orderBy = 'height:desc';
+        var params = 'orderBy=' + orderBy;
+
+        node.get('/api/peers?' + params, function (err, res) {
```

Comment:
agree, it should probably just be hard coded here since its inside the function anyways.